src/poetry/console/commands/init.py

```diff
@@ -161,10 +160,16 @@ def handle(self) -> int:
 
         python = self.option("python")
         if not python:
-            current_env = SystemEnv(Path(sys.executable))
-            default_python = "^" + ".".join(
-                str(v) for v in current_env.version_info[:2]
+            config = Config.create()
+            default_python = (
+                "^"
+                + EnvManager.get_python_version(
+                    precious=2,
```

Comment:
I stumbled a bit on the variable name `precious`. I wonder if there might be a better name or if that's a common name I just don't know?

Reply:
The two hardest things in programming are naming and caching ;) So if you have a better name I'm glad to here it.

Reply:
What about `precision`? That's what we call it when we deal with versions in poetry-core.

Reply:
I'm quite sure I wanted to name it like this :see_no_evil: :upside_down_face: